Lesson 6/src/simulation/Simulation.java

```diff
@@ -0,0 +1,178 @@
+package simulation;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+import design.Item;
+import design.Rocket;
+import design.U1;
+import design.U2;
+
+public class Simulation {
+
+	public ArrayList<Item> loadItems(String file) throws FileNotFoundException
+	{
+		ArrayList<Item> items=new ArrayList<>();
+		File phase=new File(file);
+		Scanner scanner=new Scanner(phase);
+
+		while(scanner.hasNextLine())
+		{
+			String line=scanner.nextLine();
+			String[] lineSplit=line.split("=");
+
+			Item i=new Item(lineSplit[0], Integer.valueOf(lineSplit[1]));
+			items.add(i);
+
+
+		}
+		return items;
+
+	}
+
+	public ArrayList<Rocket> loadU1 (ArrayList<Item> getItems)
+
+	{  
+
+		ArrayList<Rocket> list_rocketsU1=new ArrayList<>();
+		Rocket r1=new U1();
+		for(Item h:getItems) 
+		{
+
+			if(r1.canCarry(h))
+			{
+
+				
+				r1.carry(h);
+
+
+			}
+			else
+			{
+
+				list_rocketsU1.add(r1);
+				r1=new U1();
+				
+				
+			}
+
+		}	
+		list_rocketsU1.add(r1);
+
+		return list_rocketsU1;
+	}
+
+	public ArrayList<Rocket> loadU2 (ArrayList <Item> getItems)
+	{
+
+		ArrayList<Rocket> 	list_rocketsU2=new ArrayList<>();
+		Rocket r2=new U2();
+
+		for(Item h:getItems) {
```

Comment:
This for loop is wrong. Need to be fixed. 

Since we know the items inside phases-1&2 and U2 specs, with basic calculations we know that the rockets loaded should be:
- Phase-1: 6 Rockets
- Phase-2: 4 Rockets